### ISSUE TITLE
E2E Test : 1) "before each" hook for "should display front page slider" - fails sometimes

### ISSUE THREAD

**Issue by guttih** [island-is/island.is#2360]
When pushing new code to a pull-request in the repository, the E2E test sometimes fails.

When this happens, I need to push some comment changes in order to trigger this test again which usually changes the test result to success.

The error displayed is:
    1) "before each" hook for "should display front page slider"

  0 passing (2m)
  1 failing

  1) web "before each" hook for "should display front page slider":
     CypressError: cy.visit() failed trying to load:

**Comment by sindrig** [comment]
@guttih a link to a failed run would be great

**Comment by jeremybarbet** [comment]
Looks like web-e2e is running against our contentful data, but the cypress test is using the mock for the test.

```
cy.get('h1').contains(cms.store.frontPageSliders[0].title)
```

I've been able to run it locally successfully with 

```
API_MOCKS=true yarn e2e web-e2e --skip-cache
```

we might want to add it to the `40_e2e.sh` script or pass the API_MOCKS=true through the workspace with an environment

**Comment by guttih** [comment]
Link to a failing test: https://github.com/island-is/island.is/pull/2393/checks?check_run_id=1644920724

**Comment by saevarma** [comment]
Fixed in #2446
